### DISCUSSION TITLE
Recent changes to holiday calendar of FR/BE and CH

### DISCUSSION THREAD

**Discussion by Sushma58** [nager/Nager.Date#455] [Q&A] [answered]
Hi Team,
Greetings!
We have been using your API to get holiday calendar lists for different countries. 

I would like to know whether there are any changes implemented on 23rd Dec in holiday calendar of France, Belgium and Switzerland specifically for 26th Dec (i.e changing 26th Dec to normal working day)?

Thanks

**Comment by tinohager** [comment]
I don't understand the question.

**Comment by Sushma58** [comment]
We are using Nager Date nuget package to get the list of holidays for FRANCE. Attaching below screenshot for your reference, where it says 26th Dec 2022 is not a holiday in France.
![image](https://user-images.githubusercontent.com/78418507/209917097-ee53e0d9-ec4c-4d74-8ac8-b6fa416950b8.png)

We want to understand what is the source of data to fetch this calendar list? According to wiki 26th Dec is shown as public holiday in France however Nager date gives this date as normal working day
![image](https://user-images.githubusercontent.com/78418507/209917420-8c13da52-bb15-4186-ad34-13a9f5dfb6af.png)

**Reply by laloune** [reply]
"According to wiki 26th Dec is shown as public holiday in France however Nager date gives this date as normal working day" => this is incorrect. 26th is only suitable for 3 _départements_, 57 (Moselle), 67 (Bas-Rhin) and 68 (Haut-Rhin). 67 & 68 are contained in the FR-A iso code

the API reflects that in the counties part:
{"date":"2023-12-26","localName":"Saint-Étienne","name":"St. Stephen's Day","countryCode":"FR","fixed":true,"global":false,"counties":["FR-A","FR-57"],"launchYear":null,"types":["Public"]}

**Comment by tinohager** [comment]
**France**
https://en.wikipedia.org/wiki/Public_holidays_in_France
The two areas not states in Frances... we only support federal states.

**Switzerland**
The 26th is a holiday in certain federal states
```
{
    "date": "2023-12-26",
    "localName": "Stephanstag",
    "name": "St. Stephen's Day",
    "countryCode": "CH",
    "fixed": true,
    "global": false,
    "counties": [
      "CH-AG",
      "CH-AI",
      "CH-AR",
      "CH-BL",
      "CH-BS",
      "CH-BE",
      "CH-FR",
      "CH-GL",
      "CH-GR",
      "CH-LU",
      "CH-NW",
      "CH-OW",
      "CH-SG",
      "CH-SH",
      "CH-SZ",
      "CH-SO",
      "CH-TG",
      "CH-TI",
      "CH-UR",
      "CH-ZG",
      "CH-ZH"
    ],
    "launchYear": null,
    "types": [
      "Public"
    ]
  }
```

**Comment by Sushma58** [comment]
Thanks for the confirmation. Can you please also help me understand how frequently the holiday dates are updated?

Is it updated throughout the year as and when updates received or just once starting of the year?

**Comment by tinohager** [comment]
As soon as we receive the information, we will try to add it in a timely manner. Everyone can participate here.

**Comment by Sushma58** [comment, marked as answer]
Great thanks. Is there any API or user interface where I can check when is the holiday calendar for Switzerland and France last updated?

**Reply by tinohager** [reply]
No

**Reply by laloune** [reply]
France's public holidays are not updated very frequently (the latest was launched back in 1981)

as for Switzerland I can't tell

**Reply by tinohager** [reply]
a new endpoint is now available https://date.nager.at/api/v3/Version
